libass/ass_utils.c

```diff
@@ -504,7 +504,8 @@ int lookup_style(ASS_Track *track, char *name)
     i = track->default_style;
     ass_msg(track->library, MSGL_WARN,
             "[%p]: Warning: no style named '%s' found, using '%s'",
-            track, name, track->styles[i].Name);
+            track, name ? name : "(null)",
```

Comment:
`name` cannot be `NULL` here, it's already dereferenced earlier.

Reply:
Oh right. I even mention this in another commit message. Thanks!